.env.sample

```diff
@@ -4,3 +4,5 @@ APP_BASIC_AUTH_USERNAME=admin
 APP_BASIC_AUTH_PASSWORD=password
 APP_DATABASE_URL=postgres://torinos_db/maindb?sslmode=disable
 APP_HONEYBADGER_API_KEY=
+APP_GITHUB_CLIENT_ID=
```

Comment:
Could you also update here accordingly?
https://github.com/torinos-io/api/blob/fb46716b20c143750e51d43ceb5320efbd1ef3b6/kubernetes/deployment.yml#L56-L60